Makefile

```diff
@@ -682,15 +682,6 @@ generate_yaml_coredump: export ENABLE_COREDUMP=true
 generate_yaml_coredump:
 	$(MAKE) generate_yaml
 
-# Generate the install files, using istioctl.
```

Comment:
`istioctl gen-deploy` is deprecated and I couldn't find any place where this output was tested.